prizeout/applicant-take-home
# Checkout feature

# Description
I added functionality to the checkout panel.

1. User clicks on a gift card offer.
2. The selected gift card offer gets a blue border. The gift card shows in the checkout panel on the right. The user is given different price options.
3. The user clicks on a price option.
4. Details about the price option are shown.

# Type of Change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

1. Start the app and open a browser window at http://localhost:8069/
2. Click on a gift card offer
3. Check that the gift card offer is highlighted blue. Check that the gift card shows in the right panel along with price options. (see screenshots for reference)
4. Select a price option.
5. Check that the price details shows (see screenshots for reference)

Before:
![main-before](https://user-images.githubusercontent.com/37720729/207452414-9bc8f03d-1afc-4d66-9fef-896e980ce92f.png)
After:
![main-after](https://user-images.githubusercontent.com/37720729/207452454-c2f1e33a-c993-47fb-afe1-8ede0c988abd.png)
![main-select-amount](https://user-images.githubusercontent.com/37720729/207452499-9c97fabb-f8dc-4330-b171-b227523f1c1f.png)
![main-amount-selected](https://user-images.githubusercontent.com/37720729/207452526-b28e88b6-66ba-409d-b7f5-da3a2cebce7a.png)
